justega247/Maintenance-Tracker
# #164856892: Admin can see all requests

#### What does this PR do?
- This PR is to create a page where an admin can view all the requests in the application
#### Description of Task to be completed?
- Write code to implement the functionality
#### How should this be manually tested?
- Visit the application and login as the admin
- You should see all the requests in the application
#### What are the relevant pivotal tracker stories?
[#164856892](https://www.pivotaltracker.com/story/show/164856892)
